### PR TITLE
added co-authors

### PR DESCRIPTION
**Pull request by divyamb08**
This PR is related to "Git Co-Author - GitHub actions #892"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -734,6 +734,26 @@ For further assistance, please create an issue on the [GitHub repository](https:
 
 - Note for Contributors: If you would like to submit a Pull Request (PR), please target the `release` branch instead of `main`. The `release` branch is used for testing new code changes and will be periodically merged into `main` after validation. This approach ensures that only tested features make it into the main branch.
 
+## Git Co-Authors Feature
```
Comment (surapuramakhil) [inline README.md:737]: can you move this to contributing docs

**Review by surapuramakhil — CHANGES_REQUESTED** [review]
can you also add links of CI in action, checked repo action executed history not found.

https://github.com/divyamb08/Auto_Jobs_Applier_AI_Agent/actions/workflows/add-coauthors.yml

File: .github/workflows/add-coauthors.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          NEW_MSG="$COMMIT_MSG"$'\n\n'
+          while IFS= read -r contributor; do
+            NEW_MSG+="Co-authored-by: $contributor"$'\n'
+          done <<< "$CONTRIBUTORS"
+          
+          # Amend the merge commit with updated message
+          echo "$NEW_MSG" | git commit --amend -F -
+          
+          # Force push the changes
+          git push --force
```
Comment (surapuramakhil) [inline .github/workflows/add-coauthors.yml:50]: force pushes are restricted - it destabilizes git tree

Comment (surapuramakhil) [reply .github/workflows/add-coauthors.yml:50]: you can rely on PR approved; one PR approved trigger this CI, use GitHub anonymous email (for co-authors) print that message on PR thread.

**Comment by surapuramakhil** [comment]
@divyamb08 you can merge your hotfix branch into main and push those changes.

**Comment by divyamb08** [comment]
I made the necessary changes @surapuramakhil

**Comment by surapuramakhil** [comment]
@divyamb08 can you test your CI/CD.

checked https://github.com/divyamb08/Auto_Jobs_Applier_AI_Agent/actions/workflows/add-coauthors.yml its empty. 

can you test it
1. do some dummy commits in feature branch / PR branch
2. raise Pull request & approve that pull request (see triggered actions and its results)
3. merge Pull request, then see git log.

**Pull request closed without merging**